packages/react/.env.example

```diff
@@ -23,6 +23,7 @@ FOOTER_LOCALE_BUTTON=<Boolean flag to turn on/off the locale selector>
 BUTTON_GROUP=<Boolean flag to turn on/off the ButtonGroup component>
 CARD_LINK=<Boolean flag to turn on/off the CardLink component>
 MASTHEAD_L1=<Boolean flag to turn on/off the masthead L1>
+TABLE_OF_CONTENTS=<Boolean flag to turn on/off the TableOfContents component>
```

Comment:
rename to `DDS_TOC`, or `DDS_TABLE_OF_CONTENTS`

Reply:
updated